EventStreams/src/chronicleclient.py

```diff
@@ -0,0 +1,274 @@
+'''
```

Comment:
This probably should be under the `src` directory since it isn't compiled code. I assume that the plan is to make a module at some point?